parking/backend/user_server/wsserver.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from typing import Dict, List, Optional
+
+import attr
+from tornado import websocket
+
+import parking.shared.ws_models as models
+from parking.shared.location import Location
+
+
+class UserWSHandler(websocket.WebSocketHandler):
+    """The WebSocket handler. Handles user ws connections."""
+    def check_origin(self, origin) -> bool:
+        return True
+
+    def initialize(self, sessions_handler: 'SessionsHandler') -> None:
+        self.sh = sessions_handler
+
+    def open(self, user_id: str) -> None:
+        self.user_id = user_id
+        logging.info("WebSocket opened for user_id = '{}'".format(self.user_id))
+        self.sh.add_user(user_id, self)
+
+    def on_message(self, message: str) -> None:
+        logging.debug("Received message from user_id = '{}' : '{}'".format(self.user_id, message))
+        msg = models.deserialize_ws_message(message)
+
+        if isinstance(msg, models.LocationUpdateMessage):
+            logging.debug("Received location update from user_id = '{}'".format(self.user_id))
+            success: bool = self.sh.update_user_location(self.user_id, msg.location)
+            if not success:
+                logging.warning("SessionHandler does not hold user_id = '{}'".format(self.user_id))
+                self.close()
+        elif isinstance(msg, models.ParkingRequestMessage):
```

Comment:
We should probably make these `raise NotImplementedError` for now, so it's clear they _will_ do something in future.

Reply:
Raising exception will auto-close the connection. Probably don't want this. Let it just be like that for now.